builtin/providers/test/resource_test.go

```diff
@@ -1086,3 +1086,85 @@ resource "test_resource" "foo" {
 		},
 	})
 }
+
+func TestResource_ignoreChangesIndex(t *testing.T) {
```

Comment:
Another comment or two would help me understand what this is testing/I'm having trouble parsing the what's happening in these test steps

Reply:
Added a quick comment here. The test is only to exercise the `[0]` path syntax within `ignore_changes`.

Reply:
Helpful! Thank you!